src/main/java/edu/njit/jerse/ashe/services/MethodReplacementService.java

```diff
@@ -225,8 +224,14 @@ private static ClassOrInterfaceDeclaration getClassDeclaration(
       CompilationUnit cu, String className) throws IllegalArgumentException {
     List<ClassOrInterfaceDeclaration> classes = cu.findAll(ClassOrInterfaceDeclaration.class);
 
+    String packageName = cu.getPackageDeclaration().map(x -> x.getNameAsString()).orElse("");
```

Comment:
Is the `map` call really necessary here? I think you can you `Optional#getOrDefault` and achieve the same result.

Reply:
These lines have been removed (no longer needed since the functionality was moved)

Reply:
> also there is no getOrDefault for optionals

You're right, I was thinking of `Optional#orElse` - it's `Map` that has a `getOrDefault` method. Still, the point stands wherever this code was moved to.

Reply:
The new version is in JavaCodeCorrector#fullyQualifiedClassReference 
(as part of a loop traversing up the source tree):
```
compilationUnit
            .getPackageDeclaration()
            .ifPresent(pkg -> packageAndClassName.insert(0, pkg.getNameAsString() + "."));
```